templates/templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json

```diff
@@ -34,6 +34,14 @@
       "description": "The target framework for the project.",
       "datatype": "choice",
       "choices": [
+        {
+          "choice": "net10.0",
+          "description": ".NET 10"
```

Comment:
.NET 10 is not even in Preview 1 yet so we don't have to update the templates right now. But on the other hand if we don't do it now, we may simply forget about it. So let's merge it in the current form